charts/scalardb/values.yaml

```diff
@@ -0,0 +1,109 @@
+# Default values for scalardb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# nameOverride -- String to partially override scalardb.fullname template (will maintain the release name)
+nameOverride: ""
+# fullnameOverride -- String to fully override scalardb.fullname template
+fullnameOverride: ""
+
+scalardb:
+  # replicaCount -- Default values for number of replicas.
+  replicaCount: 3
+
+  # scalardb.contactPoints -- The database contanct point such as a hostname of Cassandra or a URL of Cosmos DB account.
+  contactPoints: cassandra
+  # scalardb.contactPort -- The database port number.
+  contactPort: 9042
+  # scalardb.username -- The username of the database. For Cosmos DB please leave blank. For Dynamo DB please specify key id here.
+  username: cassandra
+  # scalardb.password -- The password of the database. For Cosmos DB, Dynamo DB please specify a secret key here.
+  password: cassandra
+  # scalardb.storage -- Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set.
+  storage: cassandra
```

Comment:
Maybe better to define these in block of `scalardbConfiguration` like scalardl.
https://github.com/scalar-labs/helm-charts/blob/eeef41e69ebe5d2a117049a76aa2729f33ff33b2/charts/scalardl/values.yaml#L108

Reply:
I see. I'll make a block with the name `scalardbStorageConfiguration`